Aptoide/2048
# Feature/eskl 118 time up

**What does this PR do?**
Time up message when user looses due to time up.

**Database changed?**

 No

**Where should the reviewer start?**

**How should this be manually tested?**

Reduce the timeout time to about 10 seconds,
Play a game with a bot a wait for the time out. 
Check the end game message.
**What are the relevant tickets?**

  Tickets related to this pull-request: [ESKL-118](https://aptoide.atlassian.net/browse/ESKL-118)

**Questions:**





**Code Review Checklist**
